README.md

```diff
@@ -0,0 +1,108 @@
+# RITSoft
+
+## Installation
+
+- Docker
+
+  You can find the instructions to install docker [here](https://docs.docker.com/engine/install/)
+
+- Docker compose
+
+  You can find instructions to install docker-compose [here](https://docs.docker.com/compose/install/)
+
+## Getting Started
+
+- Clone the repository
+
+  ```bash
+   git clone https://github.com/RITct/RITSoft --recurse-submodules
+  ```
+
+- Open the cloned repository folder in terminal
+
+  ```bash
+   cd RITSoft
+  ```
+
+- Start the server
+  
+  ```bash
+  docker-compose up -d
+  ```
+
+  You can see the website at [localhost](http://localhost)
+
+  Use the same command to reload the server after you have made changes.
```

Comment:
Use the same command to reload the server after you have made changes to the **docker-compose.yml**. If you changed **Dockerfile** do `docker-compose up -d --build` to rebuild the image.